@xen-orchestra/xapi/src/vm.js

```diff
@@ -10,8 +10,11 @@ const { createLogger } = require('@xen-orchestra/log')
 const { Ref } = require('xen-api')
 
 const extractOpaqueRef = require('./_extractOpaqueRef')
+const isDefaultTemplate = require('./isDefaultTemplate')
 const isVmRunning = require('./_isVmRunning')
 
+const { incorrectState } = require('../../../packages/xo-common/api-errors')
```

Comment:
You cannot require across packages, you need to require them as external dependencies
.